test/AutoDiff/Inputs/derivative_registration/module2/module2.swift

```diff
@@ -0,0 +1,19 @@
+import module1
+
+extension FunctionInModule1_InternalDerivatives {
+  // TODO(TF-XXXX): This causes duplicate symbol linker errors.
+  // TODO(TF-XXXX): Why is @usableFromInline necessary?
```

Comment:
`@usableFromInline` isn't necessary if we are registering an internal derivative, but is necessary if we are registering a public derivative.